prediction_market_agent/agents/abstract.py

```diff
@@ -5,10 +5,7 @@
 
 
 class AbstractAgent:
-    def __init__(self):
-        pass
-
-    def pick_market(self, markets: list[MarketVar]) -> MarketVar:
+    def pick_market(self, markets: t.Sequence[MarketVar]) -> MarketVar:
```

Comment:
why this change?

Reply:
Just to make Mypy happy. The sequence is immutable, but I think using `list` should be OK in this case. But Mypy doesn't believe so 😄